namespaces/live.cloud-platform.service.justice.gov.uk/lucas-cp-app/01-rbac.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: lucas-cp-app-admin
+  namespace: lucas-cp-app
+subjects:
+  - kind: Group
+    name: "github:cdpt"
```

Comment:
The check fails as the github team `cdpt` doesnt exists. Can you please update with the correct github team which you are a member of